### ISSUE TITLE
options parameter missing for copy and copySync functions  in  fs-extra.d.ts

### ISSUE THREAD

**Issue by clcgithub** [soywiz/typescript-node-definitions#53]
Hi,

the fs-extra's copy and copySync functions support options parameter that is necessary while keeping the target file's time stamps after coping.

'''
copy(src, dest, [options], callback)

Copy a file or directory. The directory can have contents. Like cp -r.

Options:

clobber (boolean): overwrite existing file or directory
preserveTimestamps (boolean): will set last modification and access times to the ones of the original source files, default is false.
filter: Function or RegExp to filter copied files. If function, return true to include, false to exclude. If RegExp, same as function, where filter is filter.test.

'''

Could someone help to add this function?

**Comment by midknight41** [comment]
I don't think this repo is active anymore. All the files here were merged with https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/fs-extra/fs-extra.d.ts a while back. fs-extra's definitions have been enhanced some since that happened.

I'd suggest you submit a PR with the changes to the DefinitelyTyped repo. I'd be happy to review it.
